ledger/spverificationtracker.go

```diff
@@ -68,33 +64,27 @@ type stateProofVerificationTracker struct {
 	// log copied from ledger
 	log logging.Logger
 
+	l ledgerForTracker
+
 	// lastLookedUpVerificationContext should store the last verification context that was looked up.
 	lastLookedUpVerificationContext ledgercore.StateProofVerificationContext
 }
 
-func (spt *stateProofVerificationTracker) loadFromDisk(l ledgerForTracker, _ basics.Round) error {
-	preparedDbQueries, err := store.StateProofVerificationInitDbQueries(l.trackerDB().Rdb.Handle)
-	if err != nil {
-		return err
-	}
-
-	spt.dbQueries = preparedDbQueries
-
+func (spt *spVerificationTracker) loadFromDisk(l ledgerForTracker, _ basics.Round) error {
 	spt.log = l.trackerLog()
+	spt.l = l
 
 	spt.mu.Lock()
 	defer spt.mu.Unlock()
 
-	spt.lastLookedUpVerificationContext = ledgercore.StateProofVerificationContext{}
```

Comment:
why it does not reset lastLookedUpVerificationContext anymore?

Reply:
was removed due to :
https://github.com/algorand/go-algorand/pull/4803#discussion_r1088346054

I will restore it.